src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -827,8 +857,14 @@ private void loadProgramFailed(final Throwable problem, SettableFuture<CpsThread
     /** Report a fatal error in the VM. */
     void croak(Throwable t) {
         setResult(Result.FAILURE);
+        done = true;
```

Comment:
Why `Boolean.TRUE` above but `true` here? Might be handy to mention why you're using one vs the other in one case (probably the `Boolean.TRUE` case, since that's not something you see as often).